.travis.yml

```diff
@@ -45,16 +44,10 @@ before_script:
 
 script:
   - make
-  # Here sudo -E env "PATH=$PATH" make test is required for running tests with
-  # sudo permissions since we are making use of scsi device mockdata in smartprobe_test
-  # which could be fetched for a SCSI device with sudo permissions only since to access
-  # a particular scsi device, sudo or root permissions are required.
-  - sudo -E env "PATH=$PATH" make test
```

Comment:
We can probably leave it in here as well for a few runs.

Reply:
Sure. But the publishing report will be done only from github actions.

Reply:
Done.